engine/crates/engine-v2/Cargo.toml

```diff
@@ -10,6 +10,9 @@ readme = "README.md"
 repository = "https://github.com/grafbase/grafbase"
 version = "3.0.31"
 
+[lib]
```

Comment:
Just annoyed at `cargo test -p engine-v2` failing because cargo is trying to look for tests when they aren't any.

Reply:
Just curious here, but what kind of failure?  Literally never seen this

Reply:
if you run `cargo test -p engine-v2` it'll try to run doctest on stuff like this https://github.com/grafbase/grafbase/blob/main/engine/crates/engine-v2/src/response/key/mod.rs#L4
and fails

but also realizing that I could just wrap the relevant parts in "```text"... will remove it 😅

Reply:
You can also do ```ignore

Reply:
(or ```no_run if you want to compile it but you don't want to run it)